corehq/apps/accounting/models.py

```diff
@@ -985,6 +985,11 @@ def _process_upgrade(domain, upgraded_privileges, new_plan_version, web_user):
             raise SubscriptionChangeError("The upgrade was not successful.")
 
 
+class SubscriptionBaseManager(models.Manager):
+    def get_queryset(self):
+        return super(SubscriptionBaseManager, self).get_queryset().filter(is_hidden_to_ops=False)
```

Comment:
I don't think there's a reason from a correctness/performance perspective, just which is easier to read (which might depend on the context and also be pretty subjective)